RedHat-UX/red-hat-design-system
# docs(subnav): update best practices

## What I did

1. Updated `<rh-subnav>` "Best practices" section to a do/don't format


## Testing Instructions

1. Check [subnav's "Best practices"](https://deploy-preview-2072--red-hat-design-system.netlify.app/elements/subnavigation/guidelines/#best-practices) in DP

## Notes to Reviewers
